README.md

```diff
@@ -297,9 +332,9 @@ Add in `app/controllers/application_controller.rb`:
 
 Or if you see better fit, only in the controllers that you will be using unleash.
 
-#### Sample usage
+### 3. Sample usage
```

Comment:
I don't think this is section 3, this is general usage for the client

Reply:
Right now it's a subsection of the Rails section, that's why it's an h3, but we can make another h2 for general usage